spark/wordcount/README.md

```diff
@@ -0,0 +1,102 @@
+# Dataproc Bigtable WordCount Sample
+
+## Overview
+
+This sample demonstrates how to use  [Google Cloud Dataproc](http://cloud.google.com/dataproc), which
+provides a managed [Apache Spark](https://spark.apache.org/) environment with
+[Google Cloud Bigtable](http://cloud.google.com/bigtable/docs).
+
+This specific example provides the classic map reduce example of counting words
+in a text file.
+
+## Prerequisites
+
+1. A [Google Cloud project](console.cloud.google.com) with billing enabled. Please
```

Comment:
https://console.cloud.google.com

Otherwise, it'll be relative to current URL, and hence, a 404.